tests/ballerina-test-integration/src/test/resources/ballerinaPull/proxy-config.toml

```diff
@@ -0,0 +1,5 @@
+[proxy]
+host = "localhost"
+port = "3128"
+username = ""
+password = ""
```

Comment:
Shall we end with new line? Similarly do this for other cases.

Reply:
Fixed with  ad5c1ff125c11049a9b1c7e8ecb185e8e1e5ed02